### ISSUE TITLE
Decorated property breaks on type annotation instead of decorator (TypeScript, Babylon)

### ISSUE THREAD

**Issue by mafredri** [prettier/prettier#1996]
*Note:* Here I've used `65` as print width to demonstrate the issue.

**Input:**

```typescript
class MyContainerComponent {
  @ContentChildren(MyComponent) components: QueryList<MyComponent>;
}
```

**Output:**

```typescript
class MyContainerComponent {
  @ContentChildren(MyComponent) components: QueryList<
    MyComponent
  >;
}
```

**Expected:**

```typescript
class MyContainerComponent {
  @ContentChildren(MyComponent)
  components: QueryList<MyComponent>;
}
```

My reasoning here is that I think keeping the type annotation whole should take precedence over the decorator statement. Further, this is the behavior we would see if we added another decorator:

```typescript
class MyContainerComponent {
  @Other()
  @ContentChildren(MyComponent)
  components: QueryList<MyComponent>;
}
```

I also noticed there's yet another behavior if we decrease print width to `50`:

```typescript
class MyContainerComponent {
  @ContentChildren(
    MyComponent
  ) components: QueryList<MyComponent>;
}
```

To me this doesn't seem consistent with the behavior if we, for example, add another decorator. I believe giving a lower cost to splitting between decorator and property should improve consistency.

Reproduction link can be found [here](https://prettier.github.io/prettier/#%7B%22content%22%3A%22class%20MyContainerComponent%20%7B%5Cn%20%20%40ContentChildren(MyComponent)%20components%3A%20QueryList%3CMyComponent%3E%3B%5Cn%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A65%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D).

**Comment by vjeux** [comment]
Thanks, we should be able to improve this indeed!

**Comment by vjeux** [comment]
I just published 1.4.3 which fixes it. Thanks for reporting it!
